tutorials/text/annotations.py

```diff
@@ -94,18 +117,38 @@
 # annotations, including fancy arrows, see :ref:`plotting-guide-annotation`
 # and :doc:`/gallery/text_labels_and_annotations/annotation_demo`.
 #
+# .. _annotations-offset-text:
 #
-# Do not proceed unless you have already read :ref:`annotations-tutorial`,
-# :func:`~matplotlib.pyplot.text` and :func:`~matplotlib.pyplot.annotate`!
-#
+# Placing text annotations relative to data
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Annotations can be positioned at a relative offset to the *xy* input to
+# annotation by setting the *textcoords* keyword argument to 'offset points' or
+# 'offset pixels'.
+
+fig, ax = plt.subplots()
+x = [1, 3, 5, 7, 9]
+y = [2, 4, 6, 8, 10]
+annotations = ["A", "B", "C", "D", "E"]
+ax.scatter(x, y, s=20)
+
+for xi, yi, text in zip(x, y, annotations):
+    ax.annotate(text,
+                xy=(xi, yi), xycoords='data',
+                xytext=(1.5, 1.5), textcoords='offset points')
+
+###############################################################################
+# The annotations are offset 1.5 points (1.5*1/72 inches) from the *xy* values.
 #
 # .. _plotting-guide-annotation:
 #
-# Advanced Annotations
-# --------------------
+# Advanced annotation
```

Comment:
Personally I prefer the original American style capitalization for section headings.

Reply:
honestly so do I, but this is our documented style https://matplotlib.org/devdocs/devel/documenting_mpl.html#section-formatting

Reply:
I didn't realize that was a thing. As long as it's following the rules, I rescind my comment.